core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -605,13 +607,13 @@ private[spark] class SparkSubmit extends Logging {
 
       // Propagate attributes for dependency resolution at the driver side
       OptionAssigner(args.packages, STANDALONE | MESOS | KUBERNETES,
-        CLUSTER, confKey = "spark.jars.packages"),
+        CLUSTER, confKey = config.JAR_PACKAGES.key),
```

Comment:
for all of the references in this file, there is already a wildcard import for `config`, so you should be able to reference the names directly, right?

Reply:
yes, done!